src/main/java/net/openhft/chronicle/core/io/BackgroundResourceReleaser.java

```diff
@@ -19,6 +20,8 @@
     static {
         RELEASER.setDaemon(true);
         RELEASER.start();
+
+        PriorityHook.add(99, BackgroundResourceReleaser::releasePendingResources);
```

Comment:
Hi @alamar, I was thinking the hook might stop the `RELEASER` thread gracefully, currently it interrupts and warns because I guess it doesn't know why it was interrupted, but the hook could signal to it to exit that loop and complete and it wouldn't need to warn because it knows it's terminating due to a shutdown.
The purpose of this thread is to continually run `performClose()` on any `AbstractCloseables` for which `shouldPerformCloseInBackground()` is true, so the thread probably needs to be one of the last things to terminate before a shutdown.